src/front-end/typescript/lib/pages/organization/edit/tab/team.tsx

```diff
@@ -17,19 +17,22 @@ import React from 'react';
 import { Col, Row } from 'reactstrap';
 import CAPABILITIES from 'shared/lib/data/capabilities';
 import { AffiliationMember, memberIsOwner, memberIsPending, membersHaveCapability, MembershipType } from 'shared/lib/resources/affiliation';
-import { isVendor } from 'shared/lib/resources/user';
+import { isAdmin, isVendor } from 'shared/lib/resources/user';
 import { adt, ADT, Id } from 'shared/lib/types';
 import { validateUserEmail } from 'shared/lib/validation/affiliation';
 
 type ModalId
   = ADT<'addTeamMembers'>
   | ADT<'viewTeamMember', AffiliationMember>
-  | ADT<'removeTeamMember', AffiliationMember>;
+  | ADT<'removeTeamMember', AffiliationMember>
+  | ADT<'approveAffiliation', AffiliationMember>;
```

Comment:
I am unclear on what adding `approveAffiliation` actually does. Do anyone else understand?

Reply:
When you add a team member to a team they are in a 'pending' status.  Normally this can only be changed to 'active' by the new team member going through the accept invitation workflow.  `approveAffiliation` allows the admin (and only an admin) to bypass that workflow for new team members. It creates a button only visible to the admin that allows the removal of the pending status.